example/fcn-xs/README.md

```diff
@@ -25,6 +35,11 @@ this is the fully convolution style of the origin
 ```JPEGImages folder```, ```SegmentationClass folder```, ```train.lst```, ```val.lst```, ```test.lst```
 
 #### step2: train fcn-xs model
+* Configure GPU/CPU for training in `fcn_xs.py`.
+```python
+# ctx = mx.cpu(0)
+ctx = mx.gpu(0)
+```
```

Comment:
would you put the device as a argument to parser ? such as `--gpu`, and let -1 means cpu, 0,1,.. means the corresponding gpu?